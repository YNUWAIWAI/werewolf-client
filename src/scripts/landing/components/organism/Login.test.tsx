import * as React from 'react'
import {Login} from './Login'
import {shallow} from 'enzyme'

test('render', () => {
  const wrapper = shallow(
    <Login
      action="/login"
      csrfToken="token"
      error={false}
    />
  )

  expect(wrapper.exists()).toBe(true)
})
