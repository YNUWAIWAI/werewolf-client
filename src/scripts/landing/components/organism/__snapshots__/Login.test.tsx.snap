// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `"<form action=\\"/login\\" class=\\"la--form\\" method=\\"POST\\"><div class=\\"la--form--field\\"><label class=\\"la--form--label\\" for=\\"login-email\\">Email address</label><input autocomplete=\\"off\\" class=\\"la--form--input\\" id=\\"login-email\\" minlength=\\"6\\" name=\\"email\\" pattern=\\"[^\\\\s]*@[^\\\\s]*\\" required=\\"\\" type=\\"text\\" value=\\"\\"></div><div class=\\"la--form--field\\"><label class=\\"la--form--label\\" for=\\"login-password\\">Password</label><input autocomplete=\\"off\\" class=\\"la--form--input\\" id=\\"login-password\\" minlength=\\"8\\" name=\\"password\\" required=\\"\\" type=\\"password\\" value=\\"\\"></div><button class=\\"la--form--button\\" disabled=\\"\\" type=\\"submit\\">Login</button><input name=\\"csrfToken\\" type=\\"hidden\\" value=\\"token\\"></form>"`;
